inginious/frontend/plugins/register_students/static/email_course_registration_template.html

```diff
@@ -0,0 +1,30 @@
+<html>
+    <head></head>
+    <body>
+        <h2>$:_("Welcome on {course_name}")</h2>
+        
+        <h4>$:_("Hello {username}")</h4>
```

Comment:
There is no need to translate the parameters in template, just translate text messages and leave the parameters outside the parentheses